README.md

```diff
@@ -1,15 +1,15 @@
 # Recursion Problems
 
 ## Definitions
-Define the following: 
+Define the following:
 
-- Recursion
-- Recursive Case
-- Base Case
-- Activation Chain/Stack
-- Activation Record/Call
-- Infinite Recursion/Stack Overflow/Stack too deep
-- Tail Recursion
+- Recursion: A method that calls itself.
+- Recursive Case: The smaller problem that calls itself till the base case is hit.
```

Comment:
I would remove "till the base case is hit" only because you don't technically have to hit a base case.